deps/patches/libunwind-missing-parameter-names.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/src/ptrace/_UPT_ptrauth_insn_mask.c b/src/ptrace/_UPT_ptrauth_insn_mask.c
+index 1111111..2222222 100644
+--- a/src/ptrace/_UPT_ptrauth_insn_mask.c
++++ b/src/ptrace/_UPT_ptrauth_insn_mask.c
+@@ -50,7 +50,7 @@ unw_word_t _UPT_ptrauth_insn_mask (unw_addr_space_t as, void *arg)
+   return mask;
+ }
+ #else
```

Comment:
I'm away from computer and can't try to apply the patch, but the context looks wrong: it's missing a blank line after the #else? https://github.com/libunwind/libunwind/pull/853/files#diff-68acde150ec6aa6c96c39a5934644473e97278ba1fe27653e05d98fe1d9dc37aL51